src/projects/republic/MetadataPanel.tsx

```diff
@@ -53,7 +53,7 @@ export const MetadataPanel = (props: RenderMetadataPanelProps) => {
       ) : (
         <div>No panel defined for this annotation type.</div>
       )}
-      <AttendantsMetadata annotations={props.annotations} />
+      <DelegatesMetadata annotations={props.annotations} />
```

Comment:
Kunnen omgezet worden in labels

Reply:
Dit zit inmiddels al in `main` dmv een PR van jou.